browser/src/control/jsdialog/Widget.MenuButton.js

```diff
@@ -146,6 +146,8 @@ function _menubuttonControl (parentContainer, data, builder) {
 		if (data.image) {
 			var image = L.DomUtil.create('img', '', button);
 			image.src = data.image;
+			if (button.id.indexOf('color') !== -1)
```

Comment:
Please do not hardcode anything based on the ID of a widget.

Also, this only takes into account one case... when `color` is present. `alt` attribute should be always set, because browsers will warn about that in a11y check.

Please do this unconditionally.